### ISSUE TITLE
Implement @vercel/cache

### ISSUE THREAD

**Issue by oneseb** [nitrojs/nitro#1272]
### Describe the feature

Hi there,

we deploy our nuxt3 apps to vercel and would like to utilize their new distributed application cache. Vercel implemented an sdk that could now be implemented by the nitro team to support caching on vercel. That would abstract the complexity away from the average dev and streamline the experience.

@vercel/cache will **soon**, so not yet, be released. So how about reaching out to the vercel developers beforehand to integrate as early as possible?

Thanks in advance and kind regards

P.S. I could probably provide a contact to vercel if that is helpful.

### Additional information

- [ ] Would you be willing to help implement this feature?

**Comment by lilnasy** [comment]
Has vercel expressed interest in `@vercel/cache`?

**Comment by oneseb** [comment]
Yes they are big in the Next.js development but have many Nuxt.js customers like us. Vercel suggested they would like to invest more into the Nuxt community to further satisfy their Nuxt customers.

**Comment by pi0** [comment]
Link to docs/spec?

**Comment by RihanArfan** [comment]
We have support for Vercel Runtime Cache via unstorage https://unstorage.unjs.io/drivers/vercel#vercel-runtime-cache
